lte/gateway/c/oai/tasks/amf/Registration.cpp

```diff
@@ -652,23 +656,21 @@ static int registration_accept_t3550_handler(
       OAILOG_ERROR(
           LOG_AMF_APP,
           "Timer: timer has expired Sending Registration request again\n");
```

Comment:
do you want to indicate timer ID or timer T3XXX in the context of UE XXXX expired? is this for registration Request? can you double check? is AMF sending registration request in this case?

Reply:
Updated the log statement accordingly